### DISCUSSION TITLE
推理的最小显存估计是多少？

### DISCUSSION THREAD

**Discussion by foobra** [zai-org/ChatGLM3#984] [Q&A] [answered]
我使用hf的估算工具，FP16推理所需最小显存是5.81G （ https://huggingface.co/spaces/hf-accelerate/model-memory-usage ）
<img width="1504" alt="image" src="https://github.com/THUDM/ChatGLM3/assets/7357530/2574d2e2-e99c-4f0b-ae1e-8f689ce20a4c">

官方Readme建议是13G
<img width="863" alt="image" src="https://github.com/THUDM/ChatGLM3/assets/7357530/35638078-b2ff-410f-8132-68f792eea0e4">

这个中间的差距，主要是什么占用呢？

**Comment by zRzRzRzRzRzRzR** [comment, marked as answer]
13G，5G是int4，另外你那个链接我也打不开
